server/settings/simple_creators.py

```diff
@@ -1,13 +1,16 @@
+from pydantic import root_validator
+
 from ayon_server.settings import BaseSettingsModel, SettingsField
 
 
 class SimpleCreatorPlugin(BaseSettingsModel):
-    _layout = "expanded"
-    product_type: str = SettingsField("", title="Product type")
+
+    name: str = SettingsField(title="Name", disabled=True)
```

Comment:
What is the name for?

Reply:
yes, it's to show the name at the top

Reply:
BTW `name` is also auto-validated on frontend to be unique for all items in the model. So until frontend does support all features we need, then this won't happen.